### ISSUE TITLE
Feature Request - Default Music Disc Tracks

### ISSUE THREAD

**Issue by ComradeBlaze** [fudgeu/playlist#29]
Would love to be able to listen to the music disc songs, although I don't know how feasible this is to implement. Thanks.

**Comment by DeadbushCarrot** [comment]
He has this plan but you will have to play it in q jukebox first then it will work but thats what hes future plans are maybe am a bit wrong but I thats what I think!

**Comment by fudgeu** [comment]
Correct, I don't have an exact idea in mind yet but I don't want to make it so that you have every disc unlocked instantly, defeating the purpose of discs in the game (other than maybe in creative mode).

It's definitely planned in the future though, and maybe in the meanwhile I'll just add a little toggle to enable it for those that don't care about collecting discs :)
